borgcron.sh

```diff
@@ -144,10 +144,10 @@ for i in $REPEAT_NUMS; do
 	# exit on non-critical errors (ignore 1 = warnings)
 	if [ ${errorcode} -le 1 ]; then
 		# save/update last backup time
-		date +'%s' > "$LAST_BACKUP_DIR/$BACKUP_NAME.time"
 		if [ ! -d $LAST_BACKUP_DIR ]; then
 		mkdir -p $LAST_BACKUP_DIR
 		fi
+		date +'%s' > "$LAST_BACKUP_DIR/$BACKUP_NAME.time"
```

Comment:
I think it's correct the way it is:
1. IF directory doesn't exist THEN create dir ENDIF
2. write time in file